config/software/openssl.rb

```diff
@@ -94,11 +98,6 @@
       "/bin/bash ./Configure solaris64-x86_64-gcc -static-libgcc"
     elsif omnios?
       "/bin/bash ./Configure solaris-x86-gcc"
```

Comment:
Note, a lot of this file came originally from Chef's upstreamin omnibus-software repository. If we delete stuff like this, it's usually nicer to sync with upstream.

Reply:
Ah, I see this is a sync with upstream!